lib/screens/settings_screen.dart

```diff
@@ -19,29 +15,11 @@ class SettingsScreen extends StatefulWidget {
 class _SettingsScreenState extends State<SettingsScreen> {
   User? get user => FirebaseAuth.instance.currentUser;
   final InAppReview _inAppReview = InAppReview.instance;
-  Availability _availability = Availability.loading;
 
   @override
-    void initState() {
-      super.initState();
-
-      WidgetsBinding.instance!.addPostFrameCallback((_) async {
-        try {
-          final isAvailable = await _inAppReview.isAvailable();
-
-          setState(() {
-            // This plugin cannot be tested on Android by installing your app
-            // locally. See https://github.com/britannio/in_app_review#testing for
-            // more information.
-            _availability = isAvailable && !Platform.isAndroid
-                ? Availability.available
-                : Availability.unavailable;
-          });
-        } catch (e) {
-          setState(() => _availability = Availability.unavailable);
-        }
-      });
-    }
+  void initState() {
+    super.initState();
+  }
```

Comment:
This override does nothing. You can remove it and `super.initState` will be implicitly called  anyways